### DISCUSSION TITLE
Avoid returning null field in JSON result

### DISCUSSION THREAD

**Discussion by chipilov** [PostgREST/postgrest#2041] [Q&A] [answered]
Is it possible to configure a PostgREST call to exclude any fields with null values in the response from a query? 

For example, assume a query returns 2 columns: id and name. Assume that some rows have null value for 'name'. In this case, is it possible to get a response like this:

```
[
 { id: 1, name: 'John' }, 
 { id: 2 }
]
```
instead of this:
```
[
 { id: 1, name: 'John' },
 { id: 2, name: null }
]
```

Is this possible?

**Comment by wolfgangwalther** [comment, marked as answer]
This has been asked for in #1601, but not implemented, yet.

**Reply by chipilov** [reply]
Ok, thanks for the quick response.
